composer.json

```diff
@@ -203,9 +197,6 @@
         "symfony-web-dir": "web",
```

Comment:
This is wrong.

Reply:
It'll be removed after adding Symfony Flex and removing SensioDistributionBundle in the next PR, I'll change it to `public` before that.

Reply:
But why in a separate PR?

Reply:
This one is already +500 -800, the diff is getting unreadable and I want to make tests passing before proceeding further. It's `master` so there's still over one and a half month until the next release.

Reply:
Nevertheless, Symfony 4 config structure and Symfony Flex are two different changes, that could go separately.